docs-src/ref-programs.scrbl

```diff
@@ -1899,8 +1899,7 @@ square root up to @tt{580} squared, or @tt{336,400}.
 
 @index{pow} @reachin{pow(base, power, precision)} Calculates the approximate value of raising base to power.
 The third argument must be an @reachin{UInt} whose value is known at compile time, which represents the number
-of iterations the algorithm should perform. For reference, @tt{10} iterations provides accuracy until the
-point of overflow.
+of iterations the algorithm should perform.
```

Comment:
I wasn't saying to remove it entirely... just to add more context and give examples.